File: PRAK301/PRAK301-2210817310015-RIDHANISETIADI.c
#include<stdio.h>
int main()
{
    int A, B, C, pertama, kedua, ketiga;
    printf("Masukkan bilangan pertama :");
    scanf("%i",&A);
    printf("Masukkan bilangan kedua :");
    scanf("%i",&B);
    printf("Masukkan bilangan ketiga :");
    scanf("%i",&C);
    if(A<B){
        if(B<C){
            pertama=A;
            kedua=B;
            ketiga=C;
        }else{
            if(A<C){
                pertama=A;
                kedua=C;
                ketiga=B;
            }else{
                pertama=C;
                kedua=A;
                ketiga=B;
            }
        }
    }else{
        if(B<C){
            if(A<C){
                pertama=B;
                kedua=A;
                ketiga=C;
            }else{
                pertama=B;
                kedua=C;
                ketiga=A;
            }             
        }else{
            if(C<A){
                if(A<B){
                    pertama=C;
                    kedua=A;
                    ketiga=B;
                }else{
                    pertama=C;
                    kedua=B;
                    ketiga=A;
                }
            }
        }
    }
    printf("%i,%i,%i",pertama,kedua,ketiga);
    return 0;
}

File: PRAK301/PRAK301-2210817310015-RIDHANISETIADI.py
p=int(input("bilangan pertama = "))
q=int(input("bilangan kedua ="))
r=int(input("bilangan ketiga ="))
if p<q and p<r:
    if q<r:
        print(p,q,r)
    else:
        print(p,q,r)
elif q<p and q<r:
    if q<r:
        print(q,p,r)
    else:
        print(q,r,p)
else:
    if p<q:
        print(r,p,q)
    else:
        print(r,q,p)

        


